jinahub/indexers/searcher/keyvalue/PostgreSQLSearcher/__init__.py

```diff
@@ -33,11 +32,14 @@ def __init__(
         password: str = '123456',
         database: str = 'postgres',
         table: str = 'default_table',
+        default_traversal_paths: List[str] = ['m'],
```

Comment:
why this is `m`?

Reply:
it is a KV searcher, so its default search is for the metadata of a match from the vector searcher